Swift/ChartboostExample.xcodeproj/project.pbxproj

```diff
@@ -295,15 +334,15 @@
 					"@executable_path/Frameworks",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.chartboost.ChartboostExampleSwift;
-				PRODUCT_NAME = Chartboost;
+				PRODUCT_NAME = "$(TARGET_NAME)";
```

Comment:
I don't think renaming the app makes sense here.
The Obj-C app is called "Chartboost", and "ChartboostExample", "ChartboostExample-Pods" are not great names and don't fit in the iPhone dashboard.